integrated_channels/degreed/exporters/learner_data.py

```diff
@@ -42,13 +42,15 @@ def get_learner_data_record(
                 'degreed',
                 'DegreedLearnerDataTransmissionAudit'
             )
-            return DegreedLearnerDataTransmissionAudit(
-                enterprise_course_enrollment_id=enterprise_enrollment.id,
-                degreed_user_email=enterprise_enrollment.enterprise_customer_user.user_email,
-                course_id=enterprise_enrollment.course_id,
-                course_completed=completed_date is not None and is_passing,
-                completed_timestamp=completed_timestamp,
-            )
+            return [
+                DegreedLearnerDataTransmissionAudit(
```

Comment:
I'm trying to decide if we should do this for Degreed as well.  We're transmitting course-level content items to Degreed, so I'm assuming that we will also want to transmit a course-level completion item.

Reply:
I don't really have enough understanding of how Degreed's api behaves to say if we should do this now or not... but I suppose it makes sense.

Reply:
Yeah the more I think about it, we should be transmitting the course-level completion item to Degreed vs. the run-level completion item.